### PR TITLE
[CM T1] python binding: added .pop() .cut(), .head() and .tail() examples to keySet example

### PR DESCRIPTION
**Pull request by leothetryhard**
## Basics

- [x] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [x] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - [ ] add a line in `doc/news/_preparation_next_release.md`
  - [ ] reformat the code with `scripts/dev/reformat-all`
  - [ ] make all unit tests pass
  - [ ] fix all memleaks
- [x] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

- [ ] Add the "work in progress" label if you do not want the PR to be reviewed yet.
- [x] Add the "ready to merge" label **if the basics are fulfilled** and no further pushes are planned by you.

Continuation of #4321 after my fork broke. 
Now rebased with master with the suggested changes applied.

### CODE REVIEW

**Review by kodebach — CHANGES_REQUESTED** [review]
The examples for `cut` are good. The others functions are actually deprecated and will be removed soon. Please remove those examples and then we can merge this.

File: src/bindings/swig/python/examples/example_keyset.py [hunk 1]
```diff
@@ -23,6 +23,30 @@
 ks1.remove("user:/key6")
 print("")
 
+print("Alternatively you can use pop")
```
[the remark this answers is not in the document]
Comment (markus2330) [reply src/bindings/swig/python/examples/example_keyset.py:26]: @leothetryhard would be great if you can add deprecated notes or even better remove it from where you read that these functions are available.

File: src/bindings/swig/python/examples/example_keyset.py [hunk 2]
```diff
@@ -67,4 +91,4 @@
 ks2[0].setMeta("foo", "changed")
 print("  KeySet1[0].getMeta('foo').value={0}".format(ks1[0].getMeta("foo").value))
 print("  KeySet2[0].getMeta('foo').value={0}".format(ks2[0].getMeta("foo").value))
-print("")
+print("")
```
[the remark this answers is not in the document]
Comment (markus2330) [reply src/bindings/swig/python/examples/example_keyset.py:94]: Yes, better to leave it.

**Review by markus2330 — COMMENTED** [review]
Nice small cut/extend examples.

**Comment by kodebach** [comment]
Please rebase, so we can merge this PR too.

**Pull request merged**